arch/x86/kvm/pkvm/pkvm.c

```diff
@@ -147,6 +148,43 @@ static int pkvm_vm_init(phys_addr_t host_kvm_pa, phys_addr_t pkvm_vm_pa)
 	return ret;
 }
 
+static void teardown_donated_memory(struct pkvm_memcache *mc, void *addr, size_t size)
+{
+	/*
+	 * The pKVM hypervisor will push the memory range [addr, addr + size)
+	 * to the memcache and donate to the host. The memory range should be
+	 * PAGE_SIZE aligned. If not, it must be a code bug.
```

Comment:
nit: this comment seems unneeded, everything it describes is clear from the code?